about.html

```diff
@@ -0,0 +1,126 @@
+</head>
+
+<body>
+    <header>
+        <div class="contact-container">
+            <div></div>
+            <div class="contact">
+                <ul class="contact-ul">
+                    <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+                    <li class="contact-li"><a href="">English</a></li>
+                    <li class="contact-li"><a href="">My Page</a></li>
+                    <li class="contact-li"><a href="">sign Up</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="header-container">
+            <div class="header-container">
+                <div class="nav-container">
+                    <div class="logo">
+                        <div>
+                            <ul>
+                                <li>
+                                    <i class="fa-solid fa-graduation-cap cap"></i>
+                                    <a class="logo-link" href="home.html">Educational Resources</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+
+                    <div class="nav">
+                        <ul class="nav-ul">
+                            <li class="navigation"><a href="home.html">Home</a></li>
+                            <li class="navigation"><a href="about.html">About</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="mobile-menu">
+                        <ul class="menu-list">
+                            <li><a class="mobile-link" href="home.html">Home</a></li>
+                            <li><a class="mobile-link" href="about.html">About</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="hamburger">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="headline-container">
+            <p class="headline-p">"Hello! scholar"</p>
+            <h1>Educational Resources Summit 2023</h1>
+            <p class="headline-summary">
+                Educational resources are used in a learning environment to help and assist with people’s development
+                and
+                learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to
+                the
+                test. Educational resources are brilliant for teachers and educators to help them deliver the best
+                quality
+                lessons.
+            </p>
+            <P class="venue">
+                Please contacts us via Email for any further questions about educational resources 2023!
+                <a href="" class="link">educationalresources@edu.ng</a>
+            </P>
+        </div>
```

Comment:
- [ ] Kindly make sure you match the design of the HERO section from the About page to the one from the template. E.g.: Fons(size, weight, line height,...) on different text, spacing between and around different elements, padding, margins,...

### Template design 
![Screenshot from 2023-01-14 21-34-40](https://user-images.githubusercontent.com/83042742/212493237-bfe53cbb-f1d1-441c-bdd4-ae2fa9eb6843.png)

### Your Design 
![Screenshot from 2023-01-14 21-41-13](https://user-images.githubusercontent.com/83042742/212493287-97f0ac3a-2e1b-44ce-a8aa-faf7c1c517a2.png)